bciers/apps/administration/tests/components/contacts/Contacts.test.tsx

```diff
@@ -58,28 +58,4 @@ describe("Contacts component", () => {
       screen.queryByText(/No operations data in database./i),
     ).not.toBeInTheDocument();
   });
-
-  it("renders the correct note message and displays `Add Contact` button for external users", async () => {
```

Comment:
These weren't relevant anymore as there is no conditional rendering in the component. However I did feel slightly weird removing them, ideally they would be tested if we did `page.tsx` tests or just through e2e/screenshot testing.